README.md

```diff
@@ -84,7 +84,13 @@ pressure, it will attempt to start uncaching regions from the biggest currently
 cached one in order down to try to free memory.  If it doesn't work then it will
 throw an error.
 
-One good region to test memory usage with is nationwide zip codes: https://data.austintexas.gov/d/a3it-2a2z with ~34000 features.
+One good region to test memory usage with is nationwide zip codes: https://data.austintexas.gov/d/a3it-2a2z with ~34000 features.  Another one is: qz3q-ghft.
```

Comment:
Can you specify if qz3q-ghft is a production dataset or in some other environment?